lib/alchemy/on_page_layout/callbacks_runner.rb

```diff
@@ -0,0 +1,38 @@
+module Alchemy
+  module OnPageLayout
+
+    # Runs OnPageLayout callbacks
+    #
+    # Included in +Alchemy::PagesController+ and +Alchemy::Admin::PagesController+
+    #
+    # @see OnPageLayout in order to learn how to define +on_page_layout+ callbacks.
+    #
+    module CallbacksRunner
+      extend ActiveSupport::Concern
+
+      included do
+        before_action :run_on_page_layout_callbacks, only: :show,
+          if: -> { OnPageLayout.callbacks.present? }
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.